src/order/complete_lattice.lean

```diff
@@ -818,6 +818,17 @@ supr_subtype
 lemma infi_subtype'' {ι} (s : set ι) (f : ι → α) : (⨅ i : s, f i) = ⨅ (t : ι) (H : t ∈ s), f t :=
 infi_subtype
 
+lemma bsupr_const {ι : Sort*} {a : α} {s : set ι} (hs : s.nonempty) :
+  (⨆ (i ∈ s), a) = a :=
+begin
+  haveI : nonempty s := set.nonempty_coe_sort.mpr hs,
```

Comment:
```suggestion
  haveI : nonempty s := hs.coe_sort,
```
You might have to merge master to get this.

Reply:
Are you sure this exists?

Reply:
Ugh, I added [`set.subsingleton.coe_sort`](https://leanprover-community.github.io/mathlib_docs/data/set/basic.html#set.subsingleton.coe_sort) but forgot `set.nonempty.coe_sort`.

Reply:
Added in #16965